Meta/toot-commits.js

```diff
@@ -0,0 +1,34 @@
+const fs = require("fs");
+const Mastodon = require("mastodon");
+const { ACCESS_TOKEN } = process.env;
+const tootLength = 500;
+// Mastodon always considers links to be 23 chars, see https://docs.joinmastodon.org/user/posting/#links
+const mastodonLinkLength = 23;
+
+const M = new Mastodon({
```

Comment:
`const mastodon = ...` - not sure if this is from their docs or something, but instances should use `camelCase` in JS.

Reply:
I just replaced the `T` from `tweet-commits.js` with `M` :shrug:

Reply:
oh, I missed that it's basically a copy/paste 😅